librz/core/tui/panels.c

```diff
@@ -4938,79 +4937,79 @@ void __panels_check_stackbase(RzCore *core) {
 
 void __init_rotate_db(RzCore *core) {
 	RzCoreVisual *visual = core->visual;
-	Sdb *db = visual->panels->rotate_db;
-	sdb_ptr_set(db, "pd", &__rotate_disasm_cb, 0);
-	sdb_ptr_set(db, "p==", &__rotate_entropy_h_cb, 0);
-	sdb_ptr_set(db, "p=", &__rotate_entropy_v_cb, 0);
-	sdb_ptr_set(db, "px", &__rotate_hexdump_cb, 0);
-	sdb_ptr_set(db, PANEL_CMD_REGISTERS, &__rotate_register_cb, 0);
-	sdb_ptr_set(db, "af", &__rotate_function_cb, 0);
-	sdb_ptr_set(db, PANEL_CMD_HEXDUMP, &__rotate_hexdump_cb, 0);
+	HtSP *db = visual->panels->rotate_db;
+	ht_sp_insert(db, "pd", &__rotate_disasm_cb);
```

Comment:
While you are on it, could you please extract that part into a `librz/core/tui/modes.c` maybe? So we have those modes for various visual modes defined there.

Reply:
I would prefer to not touch it.